### ISSUE TITLE
[BUG]: 

### ISSUE THREAD

**Issue by JanSuerth** [LSS-Manager/LSSM-V.4#2052]
### What happened?

On the US Version the missing-vehicles alert displays missing Hazmat personnel. When "Enhanced missing vehicles" from the "Extended alarm window" module is enabled the personnel also gets displayed in the table. 

When selecting a vehicle the counter for the educated hazmat personnel doesn't count people on the vehicle but rather only the hazmat vehicle (so the counter goes up by one even when "maximum staff" is enabled)

Vehicles like the Hazmat Ambulance do not get counted at all for the personnel. 

Default display: https://i.imgur.com/TU5K78s.png
With enhanced missing vehicles: https://i.imgur.com/qXprl1N.png
(In both pictures there is a HazMat ambulance selected but not yet dispatched. It is staffed with 6 educated personnel and "maximum staff" is enabled for enhanced missing vehicles.

### How to reproduce?

1. extended missing vehicles needs to be enabled
2. maximum staff needs to be enabled
3. in a mission with missing hazmat personnel select a HazMat or HazMat ambulance

### Expected behavior

I expected that the personnel of the HazMat or HazMat ambulance would be counted towards the missing personnel rather then counting the number of Hazmat vehicles selected.

### System

- OS: Windows 11
- Browser and version: Edge - Version 108.0.1462.54 (Official build) (64-bit)
- LSSM-Version: 4.7.2+20221226.1422 [stable]
- Game-Version: https://www.missionchief.com/

### Additional

![msedge_m8DvenHE7D](https://user-images.githubusercontent.com/32181822/210461232-f4eca51d-f508-4469-9620-cef35913f299.png)
![tnxv0UHZTo](https://user-images.githubusercontent.com/32181822/210461235-8afd77a1-e5b8-4e42-944a-b19c0851209d.png)

**Comment by Cr4zyc4k3** [comment]
@jxn-30 This should be fixable with a few tweaks in https://github.com/LSS-Manager/LSSM-V.4/blob/1d360ca05e9f370a37a7cfeb663bfcf99ee14c52/src/modules/extendedCallWindow/i18n/en_US.json#L91 or did I miss something?

**Comment by jxn-30** [comment]
> @jxn-30 This should be fixable with a few tweaks in https://github.com/LSS-Manager/LSSM-V.4/blob/1d360ca05e9f370a37a7cfeb663bfcf99ee14c52/src/modules/extendedCallWindow/i18n/en_US.json#L91 or did I miss something?

I don't think so. The parser will still count the text as a vehicle (there is no fully reliable way to determine for a text if it is a vehicle or a training requirement). As the texts are exactly same for HazMat vehicle and training requirement, the issue is caused.

**Comment by Cr4zyc4k3** [comment]
Next try: With https://forum.leitstellenspiel.de/index.php?thread/8406-infos-f%C3%BCr-entwickler/&postID=495288#post495288
we should be able to distinguish between personal, vehicles and "others"

**Comment by Cr4zyc4k3** [comment]
@JanSuerth Is this fixed now?
